### PR TITLE
CM-1020 - Clean up and add sovrn

### PR DESCRIPTION
**Pull request by 3link**
Related PRs: 
- https://github.com/LiveIntent/idres/pull/1411
- https://github.com/LiveIntent/Prebid.js/pull/26
- https://github.com/LiveIntent/castle/pull/449

### CODE REVIEW

File: dev-docs/bidders/criteo.md [hunk 1]
```diff
@@ -109,6 +109,18 @@ var adUnits = [
 
 ### Additional Config (Optional)
 
+If you don't want to use the FastBid adapter feature, you can disable it via this configuration:
```
Comment (mschuwalow) [inline dev-docs/bidders/criteo.md]: This seems unrelated

File: dev-docs/bidders/indiecue.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
```
Comment (mschuwalow) [inline dev-docs/bidders/indiecue.md]: also unreleated

File: dev-docs/bidders/oms.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (mschuwalow) [inline dev-docs/bidders/oms.md]: same

**Review by mschuwalow — APPROVED** [review]

File: dev-docs/bidders/criteo.md (hunk 1 above)
Comment (3link) [reply dev-docs/bidders/criteo.md]: Good catch! Some branches hick-up. Fixed now.

**Review by peixunzhang — APPROVED** [review]

**Comment by 3link** [comment]
Opened a PR in Prebid's repo: https://github.com/prebid/prebid.github.io/pull/5006
